React/wbooks/src/Services/UserService.js

```diff
@@ -1,7 +1,7 @@
 import api from '../Config/api';
 
 const UserService = {
-  login: async (email, password) => api.post('/users/sessions', { email, password }),
+  login: async user => api.post('/users/sessions', { ...user }),
```

Comment:
This shouldn't need to be a copy of `user` (i.e. `...user`), we can pass it directly in the second parameter 😄